.editorconfig

```
@@ -3,7 +3,7 @@ root = true
 
 [*]
 indent_style = space
-indent_size = 4
+indent_size = 2
```

Rust code style uses 4-spaces indentation.

```suggestion
indent_size = 4
```